app/views/steps/applicant/contact_details/edit.html.erb

```diff
@@ -1,17 +1,21 @@
 <% title t('.page_title') %>
+<% step_header %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <%= step_header %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary %>
 
-    <h1 class="heading-xlarge gv-u-heading-xxlarge"><%=t '.heading', name: @form_object.record.full_name %></h1>
+    <h1 class="govuk-heading-xl"><%=t '.heading', name: @form_object.record.full_name %></h1>
 
-    <%= step_form @form_object do |f| %>
-      <%= f.text_field :email, class: 'narrow' %>
-      <%= f.text_field :home_phone %>
-      <%= f.text_field :mobile_phone %>
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_email_field :email, label: { size: 's' } %>
+      <%= f.govuk_text_field :home_phone, label: { size: 's' } %>
```

Comment:
Let's use for phone fields, when possible, `govuk_phone_field`:
https://govuk-form-builder.netlify.app/form-elements/text-field/